routers/users.py

```
@@ -16,242 +17,49 @@
-        raise HTTPException(status_code=401,detail="없는 유저입니다.")
-
-    if not user_record or user_record["password"] != login_data.password:
-        print(f"DB비번: {user_record['password']}, 입력비번: {login_data.password}")
-        raise HTTPException(status_code=401, detail="인증 정보가 올바르지 않습니다.")
-
-    token, expires_in = create_access_token(user_record["user_id"])
-
-    return UserResponse(
+@router.post("/auth/login",response_model=AuthResponse)
+async def login(data: Annotated[UserLogin, Depends(OAuth2PasswordRequestForm)],
+    form_data: OAuth2PasswordRequestForm = Depends()):
```

Ruff (0.14.11)

Unused function argument: `form_data`

(ARG001)